quantum/backlight/backlight_arm.c

```diff
@@ -0,0 +1,261 @@
+#include "quantum.h"
+#include "backlight.h"
+#include <hal.h>
+#include "debug.h"
+
+// TODO: remove short term bodge when refactoring BACKLIGHT_CUSTOM_DRIVER out
+#ifdef BACKLIGHT_PIN 
+
+#if defined(STM32F1XX) || defined(STM32F1xx)
+    #define USE_GPIOV1
+#endif
+
+//GPIOV2 && GPIOV3
+#ifndef BACKLIGHT_PAL_MODE
+    #define BACKLIGHT_PAL_MODE      2
+#endif
+
+// GENERIC
+#ifndef BACKLIGHT_PWM_DRIVER
+    #define BACKLIGHT_PWM_DRIVER    PWMD3
+#endif
+#ifndef BACKLIGHT_PWM_CHANNEL
+    #define BACKLIGHT_PWM_CHANNEL    3
+#endif
+
+static void breathing_callback(PWMDriver *pwmp);
+
+static PWMConfig pwmCFG = {
+  0xFFFF,                              /* PWM clock frequency  */
+  256,                              /* PWM period (in ticks) 1S (1/10kHz=0.1mS 0.1ms*10000 ticks=1S) */
+  NULL,                               /* No Callback */
+  {
+      {PWM_OUTPUT_DISABLED, NULL}, /* Enable Channel 0 */
```

Comment:
does the comment make sense? or should it say PWM_OUTPUT_ENABLED?

Reply:
Due to the channel being variable, but the pwmCFG static, the code currently defaults to disabled and updates during init. I have updated the comment for clarification.